changelogs/unreleased/121-pawanpraka1

```diff
@@ -0,0 +1 @@
+enhancement to bound the pvc only if volume has been created
```

Comment:
n_ : bound -> bind

Reply:
Fixes an issue where ZFS PV with an invalid pool was bound to PVC.

Reply:
updated the description.